assets/scss/components/_forms.scss

```diff
@@ -149,3 +149,78 @@ Faux inputs visually mimic actual input elements.
 		}
 	}
 }
+
+$toggleSize: 20px;
+$C_toggleHighlight: $C_lightPurple; // replace $C_lightPurple with $C_blue once we start to use `swarm-constants`
+
+.toggleLabel {
+	padding-bottom: $space-half;
+
+	.toggleLabel-container {
+		padding-left: $space-half;
+	}
+
+	&:hover {
+
+		.fauxToggle {
```

Comment:
I'm more than happy to use something other than `toggle`. Any ideas what name might be good to describe both a `input type=checkbox` as well as a `input type=radio`?